haystack/document_stores/graphdb.py

```diff
@@ -49,37 +47,60 @@ def __init__(
         self.password = password
         self.prefixes = prefixes
 
-    def create_index(self, config_path: Path, headers: Optional[Dict[str, str]] = None):
+    def create_index(
+        self,
+        config_path: Path,
+        headers: Optional[Dict[str, str]] = None,
+        timeout: Union[float, Tuple[float, float]] = 10.0,
+    ):
         """
         Create a new index (also called repository) stored in the GraphDB instance
 
         :param config_path: path to a .ttl file with configuration settings, details:
         :param headers: Custom HTTP headers to pass to http client (e.g. {'Authorization': 'Basic YWRtaW46cm9vdA=='})
+        :param timeout: How many seconds to wait for the server to send data before giving up,
+            as a float, or a :ref:`(connect timeout, read timeout) <timeouts>` tuple.
+            Defaults to 10 seconds.
+        :type timeout: Union[float, Tuple[float, float]]
```

Comment:
No need to add the type here as we already have it in the type annotation in the methods signature.